### PR TITLE
Add Application CRD installation to tool prerequisites.

### PR DESCRIPTION
**Pull request by trironkk**

### CODE REVIEW

File: docs/tool-prerequisites.md [hunk 1]
```diff
@@ -47,3 +47,10 @@ mpdev /scripts/doctor.py
 ```
 
 Save your gcloud project to an environment variable.
+
+
+## Install Application CRD
```
Comment (huyhg) [inline docs/tool-prerequisites.md]: can you move this to before the doctor tool?

File: docs/tool-prerequisites.md [hunk 2]
```diff
@@ -47,3 +47,10 @@ mpdev /scripts/doctor.py
...
+
+```shell
+kubectl apply -f "https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/crd/app-crd.yaml"
```
Comment (huyhg) [inline docs/tool-prerequisites.md]: Can you update [this line](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/89bdb09b875bde5a55f1c8cf31592b7232de99b7/scripts/doctor.py#L202) to remove `curl` too?

Comment (trironkk) [reply docs/tool-prerequisites.md]: Done!

File: docs/tool-prerequisites.md (hunk 1 above)
Comment (trironkk) [reply docs/tool-prerequisites.md]: Done!

**Comment by trironkk** [comment]
Will not push until `cloudbuild.yaml` is fixed.

**Pull request merged**